woof-code/3builddistro

```diff
@@ -493,6 +493,23 @@ for i in $(ls sandbox3/rootfs-complete/usr/share/locale); do
 	case $i in en|en_US|locale.alias) continue ;; esac
 	rm -rf sandbox3/rootfs-complete/usr/share/locale/$i
 done
+
+#overwrite current documentation if DOCS_URL present
+if [ -n "$DOCS_URL" ]; then
+	echo "Overwriting current documentation with developer provided $DOCS_URL..."
+	rm -rf sandbox3/rootfs-complete/usr/share/doc
+
+	[ -f "../../local-repositories/$DOCS_URL.doc.zip" ] || wget_output=$(wget -t 3 -T 60 -O "../../local-repositories/$DOCS_URL.doc.zip" "$DOCS_URL")
+
+	if [ $wget_output -ne 0 ]; then
+		echo "ERROR: wget failed. Read above lines to see the error."
+		exit 1
+	fi
+
+	unzip doc.zip -d sandbox3/rootfs-complete/usr/share/
```

Comment:
These two lines are broken, you're downloading to one place and attempting to extract from another

Reply:
I updated it now. The earlier caching system was completely bad.